### PR TITLE
(maint) eastwood linting

### PR DESCRIPTION
**Pull request by wkalt**
I ran lein eastwood on the repo and these are the things that jumped out at me.
A few docstrings after function arguments, a few places where tests weren't
running as expected.

### CODE REVIEW

**Comment by ajroetker** [comment]
@pljenkinsro retest this please

**Comment by wkalt** [comment]
@pljenkinsro retest this please

File: test/puppetlabs/puppetdb/query/reports_test.clj [hunk 1]
```diff
@@ -205,21 +203,21 @@
...
+                       :resource-title "hi",
                        :timestamp "2014-12-24T00:01:12Z",
-                       :containment_path ["Stage[main]" "Main" "Notify[hi]"],
+                       :containment-path ["Stage[main]" "Main" "Notify[hi]"],
                        :message "defined 'message' as 'Hi world'"}]}])
 
 (deftest structured-data-seq
   (testing "structured data seq gets correct result"
-    (is (r/structured-data-seq :v4 data-seq) expected-result))
+    (is (= (r/structured-data-seq :v4 data-seq) expected-result)))
```
Comment (senior) [inline test/puppetlabs/puppetdb/query/reports_test.clj]: This is a really good catch

**Comment by senior** [comment]
This is catching some good stuff, I like it.

**Pull request merged**